edgelet/iotedge/src/check/checks/aziot_edged_version.rs

```diff
@@ -18,8 +24,113 @@ impl Checker for AziotEdgedVersion {
     fn description(&self) -> &'static str {
         "latest security daemon"
     }
-    fn execute(&mut self, check: &mut Check, _: &mut tokio::runtime::Runtime) -> CheckResult {
-        self.inner_execute(check)
+    fn execute(
+        &mut self,
+        check: &mut Check,
+        tokio_runtime: &mut tokio::runtime::Runtime,
+    ) -> CheckResult {
+        let settings = if let Some(settings) = &check.settings {
+            settings
+        } else {
+            return CheckResult::Skipped;
+        };
+
+        if settings.parent_hostname().is_some() {
+            return CheckResult::Ignored;
```

Comment:
Add a comment.

```rust
// This is a nested Edge device so it may not be able to access aka.ms or github.com.
// In the best case the request would be blocked immediately, but in the worst case it may take a long time to time out.
// The user didn't provide the expected version on the command line either. So just ignore this check.
```